finality-aleph/src/network/gossip/mod.rs

```diff
@@ -91,6 +91,8 @@ pub trait RawNetwork: Clone + Send + Sync + 'static {
     type EventStream: EventStream<Self::PeerId>;
 
     /// Returns a stream of events representing what happens on the network.
+    /// Watch out for race conditions, events that appeared in network before the stream
+    /// was created will not be reported.
     fn event_stream(&self) -> Self::EventStream;
```

Comment:
It is, and it actually should be removed, because one should never call it in `finality-aleph`, because it'll be too late. Removed this method and also `EventStream` associated type (making it an extra generic parameter of `Service`)